doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+
+![Mission Execution Use Cases](./images/Mission-Execution-Use-Cases.png)
+
+## 1.4 Architectural Components
+
+The design of the ROS 2 Navigation System will attempt to generalize the navigation system from ROS 1 and address some of its limitations. While this document does not attempt to define the ROS 2 Navigation System architecture, there are some expected modules that are convenient to call out for the purposes of defining more detailed requirements for the system. It is useful to split these modules into two general categories: those that are involved in the *command chain*, where each level refines its command input into successively more specific operations for the lext level down, and those modules that are *support modules* used by modules in the command chain. 
+
+## 1.4.1 Command Chain
+
+The command chain is the sequence of modules that comprise the chain of command from the user, at the top, to the robot, at the bottom.
+
+* **User** - The user interacts with a Mission Planning subsystem to specify a plan for the robot. 
+* **Mission Planning** - The Mission Planning module provides an interface to the user to allow the user to create mission plans. The resulting plans are available for assignment to robots and subsequent execution. The Mission Planning subsystem is outside the scope of the Navigation System. However, the data format, or messages, required to interact with the subsystem area within scope and must be specified as the Navigation System needs to receive this information to be able to execute the mission plans.
+* **Mission Execution** - The Mission Execution module receives the mission plan as input and is responsible to execute it. It can select the appropriate global and local planners to use to perform each primitive maneuver. It also reports back progress on the execution of the mission plan. 
+* **Global Planning** - The Global Planning module, of which there could be several specialied implementations, use information from the map(s) to determine a route for the robot to achieve its goal.
+* **Local Planning** - The Local Planning module, of which there could also be several specialized implementations, uses the defined route from the Global Planner and information from robot sensors to handle any required deviations from the route to achieve its goal. 
+* **Robot Interface** - The Robot Interface is an abstraction of the robot platform, providing the means to control the robot and learn about its capabilities. There should be a standard set of messages to control the robot and, perhaps, a standard way to determine a robot's capabilities and attributes. This may require a layer on top of the vendor interface.
+
+![Command Pipeline](./images/Command-Pipeline.png)
```

Comment:
Do you think this use of the turtlebot image falls under the "fair use" defense?

Reply:
I would think so. But furthermore, I think the turtlebot BDFL group would OK it either way. The Turtlebot is designed to be the onboarding point for using the core ROS apps. It depends heavily on the nav stack in ROS1, and will almost certainly depend on this package in ROS2.

Reply:
I'll look into the status of the turtlebot image as well (and any future images).